test/trento_web/plugs/api_redirector_test.exs

```diff
@@ -0,0 +1,54 @@
+defmodule TrentoWeb.Plugs.ApiRedirectorTest do
+  use TrentoWeb.ConnCase, async: true
+  use Plug.Test
+
+  alias TrentoWeb.Plugs.ApiRedirector
+
+  defmodule StubRouter do
+    def __routes__ do
+      [%{verb: :get, path: "/api/v1/test"}]
```

Comment:
I guess this handles also dynamic routes like `/api/v1/some-resource/:resource_id` by properly intercepting `/api/some-resource/12345` and redirecting to `/api/v1/some-resource/12345`.

Maybe worth adding a scenario in the tests?

Reply:
nice catch! done